### PR TITLE
Trigger beta tests on travis every day.

### PR DESCRIPTION
**Pull request by elopio**
LP: #1662975

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,16 +33,19 @@ install:
...
     -v $PWD:$PWD -w $PWD -td ubuntu:xenial; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder locale-gen en_US.UTF-8; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder sed -i s/archive.ubuntu.com/us.archive.ubuntu.com/g /etc/apt/sources.list; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder apt update; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder sh -c "$DEPENDENCIES"; fi
 
 script:
-  - if [ ! -z $CHECK_CLA ] && [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./tools/cla_check_in_travis.sh; fi
-  - if [ ! -z $TEST_SUITE ]; then docker exec -i builder ./runtests.sh $TEST_SUITE; fi
+  - if [ ! -z $CHECK_CLA ] && [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_EVENT_TYPE" != 'cron' ]; then ./tools/cla_check_in_travis.sh; fi
```
Comment (kyrofa) [inline .travis.yml]: You can probably get rid of the `$TRAVIS_PULL_REQUEST` check here, no? Now that you're checking the event type?

Comment (kyrofa) [inline .travis.yml]: Oh wait, no, ignore me, sorry. !=

Comment (elopio) [inline .travis.yml]: No, because it would run when the branch is landed in master. That's != pull request and != cron.

**Comment by elopio** [comment]
Thanks for the +1 @kyrofa. I replied to your question here, but had to move the branch to #1144 because I don't know why, travis got stuck on this one.

**Pull request closed without merging**